keyboards/ergodox_ez/keymaps/danielo515/keymap.c

```diff
@@ -0,0 +1,320 @@
+#include "ergodox_ez.h"
+#include "debug.h"
+#include "action_layer.h"
```

Comment:
These should be replaced by: 
```suggestion
#include QMK_KEYBOARD_H
```